### PR TITLE
clientv3: fix typo

### PR DESCRIPTION
**Pull request by purpleidea**
Sorry for small shitty patches. Hope to have better ones in the future, but I send it as I find it :P

### CODE REVIEW

**Comment by xiang90** [comment]
lgtm

**Comment by purpleidea** [comment]
I squeezed in another typo fix into this patch in case you haven't merged yet

File: clientv3/example_watch_test.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func ExampleWatcher_watchWithRange() {
 	}
 	defer cli.Close()
 
-	// watches within ['foo1', 'foo4'), in lexicographical order
```
Comment (gyuho) [inline clientv3/example_watch_test.go]: I believe this is correct. https://github.com/coreos/etcd/blob/master/clientv3/op.go#L195

/cc @heyitsanthony

Comment (purpleidea) [inline clientv3/example_watch_test.go]: On Fri, May 20, 2016 at 5:10 PM, Gyu-Ho Lee <notifications@github.com>
wrote:

> I believe this is correct.
> https://github.com/coreos/etcd/blob/master/clientv3/op.go#L195
>
> /cc @heyitsanthony <https://github.com/heyitsanthony>
>


I'd say that's a typo too. LMK

Comment (gyuho) [inline clientv3/example_watch_test.go]: @purpleidea Sorry, I meant to say `[a, b)` is correct.

Comment (heyitsanthony) [inline clientv3/example_watch_test.go]: It's `[a,b)`. Cf. https://github.com/coreos/etcd/blob/master/mvcc/index.go#L157

**Comment by purpleidea** [comment]
Put back original patch, although I think the [, ) format is AFAIK weird or a bug. The classical mathematical definition for ranges is [, ]

To include or exclude an end value you use ] & [, eg: [4, 6[ or ]3, 5] would both mean 4, 5 if in R.
Cheers!

File: clientv3/example_watch_test.go (hunk 1 above)
Comment (purpleidea) [inline clientv3/example_watch_test.go]: On Fri, May 20, 2016 at 5:21 PM, Anthony Romano <notifications@github.com>
wrote:

> It's [a,b). Cf.
> https://github.com/coreos/etcd/blob/master/mvcc/index.go#L157
>

I think we're looking at if the closing bracket should be round or square.

**Comment by heyitsanthony** [comment]
@purpleidea my understanding is that `],[` is Bourbaki notation; `(,)`is traditional/classical.

**Comment by purpleidea** [comment]
On Fri, May 20, 2016 at 6:09 PM, Anthony Romano <notifications@github.com>
wrote:

> @purpleidea <https://github.com/purpleidea> my understanding is that ],[
> is Bourbaki notation; (,)is traditional/classical.



Sure, except the example mixes parts from both...

**Pull request merged**